cmd/pinniped/cmd/kubeconfig.go

```diff
@@ -922,10 +922,7 @@ func selectUpstreamIDPFlow(discoveredIDPFlows []idpdiscoveryv1alpha1.IDPFlow, se
 		return discoveredIDPFlows[0], nil
 	default:
 		// The user did not specify a flow, and more than one was found.
-		return "", fmt.Errorf(
-			"multiple client flows for Supervisor upstream identity provider %q of type %q were found, "+
-				"so the --upstream-identity-provider-flow flag must be specified. "+
-				"Found these flows: %v",
-			selectedIDPName, selectedIDPType, discoveredIDPFlows)
+		log.Info("multiple client flows found, selecting first value as default: "+discoveredIDPFlows[0].String(), "idpName", selectedIDPName, "idpType", selectedIDPType)
```

Comment:
Also log all the other options for `discoveredIDPFlows`.

Reply:
We need to update

https://github.com/vmware-tanzu/pinniped/blob/24b0ddf600c2c20cfbed5e455e282957267234e1/cmd/pinniped/cmd/login_oidc.go#L273-L274

as well.